src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -52,7 +55,7 @@ const linkStyles = css`
     background: ${brandAlt[400]};
     border-radius: 16px;
     box-sizing: border-box;
-    color: ${palette.neutral[7]};
+    color: ${neutral[7]};
```

Comment:
primary?

Reply:
For pretty much [the same reasoning as before](https://github.com/guardian/dotcom-rendering/pull/1274#discussion_r394669244) I've chosen to use global colours instead of functional colours in the reader revenue links